app/controllers/CreditsSummaryController.scala

```diff
@@ -85,27 +87,37 @@ class CreditsSummaryController @Inject()(creditsView: CreditsSummary,
                     origin: Option[String] = None)
                    (implicit user: MtdItUser[AnyContent], hc: HeaderCarrier, ec: ExecutionContext): Future[Result] = {
     if (isEnabled(MFACreditsAndDebits)) {
```

Comment:
we need something urgently like scalafmt, as this is nightmare to compare and extract what changed..

Reply:
Absolutely agree